packages/evm/test/registry/guardians-registry-locking.js

```diff
@@ -32,32 +27,22 @@ contract('GuardiansRegistry', ([_, guardian, governor]) => {
     anotherLockManager = await LockManager.new(registry.address)
   })
 
-  const lockActivation = async (recipient, lockManager, amount, sender = undefined, authorize = false) => {
-    if (!sender) return lockManager.lockActivation(recipient, amount)
-    let calldata = registry.contract.methods.lockActivation(recipient, lockManager.address, amount.toString()).encodeABI()
-    if (authorize) calldata = await encodeAuthorization(registry, recipient, externalAccountPK, calldata, sender)
-    return registry.sendTransaction({ from: sender, data: calldata })
-  }
-
-  const unlockActivation = async (recipient, amount, sender = undefined, deactivate = false, authorize = false) => {
-    if (!sender) return lockManager.unlock(recipient, amount)
-    let calldata = registry.contract.methods.unlockActivation(recipient, lockManager.address, amount.toString(), deactivate).encodeABI()
-    if (authorize) calldata = await encodeAuthorization(registry, recipient, externalAccountPK, calldata, sender)
-    return registry.sendTransaction({ from: sender, data: calldata })
+  const activate = async (amount) => {
+    await ANT.generateTokens(guardian, amount)
+    await ANT.approve(registry.address, amount, { from: guardian })
+    return registry.stakeAndActivate(guardian, amount, { from: guardian })
   }
 
-  const activate = async (recipient, amount, sender = guardian) => {
-    await ANT.generateTokens(sender, amount)
-    await ANT.approve(registry.address, amount, { from: sender })
-    let calldata = registry.contract.methods.stakeAndActivate(recipient, amount.toString()).encodeABI()
-    calldata = await encodeAuthorization(registry, recipient, externalAccountPK, calldata, sender)
-    return registry.sendTransaction({ from: sender, data: calldata })
+  const lockActivation = async (lockManager, amount, sender = undefined) => {
```

Comment:
Hmm ok, so wondering, if these are really meant as optional arguments, perhaps we can use a trailing options argument to mark them as such?